src/core/experimental/memory/simd.d

```diff
@@ -0,0 +1,170 @@
+/**
```

Comment:
Should this be in `core.experimental.memory`? Wouldn't `core.experimental` make more sense?

Reply:
It depends on how you look at it. It was proposed here: https://github.com/dlang/druntime/pull/2671#discussion_r305198634
Which makes sense if the SIMD operations are only memory operations (which they are currently). Eventually, I hope that this will become a bigger SIMD library and be moved out to `experimental` (or wherever) because D needs one that is able to cross-compile. At least I don't know of one and so I made this.